app/presenters/migration/new.rb

```diff
@@ -8,7 +8,7 @@ def content
           title: page_content_heading,
           subtitle: t('contribute.campaigns.migration.pages.new.subtitle'),
           hero: {
-            url: 'https://europeana-portal-production.s3.amazonaws.com/media_objects/000/001/744/files/94235082efaf0b297ef4cc4f377fd8ed.original.jpg?1509521460'
+            url: asset_path('ugc-form-header.jpg')
```

Comment:
Could we rename all of the images for the migration campaign in app/assets/images thus:

* migration-new-hero.jpg (for the one used in this page)
* migration-index-hero.jpg
* migration-index-preview-1.jpg
* migration-index-preview-2.jpg
* migration-index-preview-3.jpg

Reply:
The open pr 99 references the same image files so this would set up a merge conflict.  I had git mangle an image (locally, when I moved ugc-preview-2.jpg to ugc-preview-1.jpg as part of that ticket).  It may be simpler to do this after both these PRs (99 & 100) have been merged.

Reply:
Rename them here, then we can merge this PR, then merge develop into #99 and resolve its conflicts.

Reply:
@andyjmaclean: alternatively, close this PR and make the changes including renaming images on #99.

Reply:
Done